src/openrct2/ReplayManager.cpp

```diff
@@ -610,7 +610,7 @@ namespace OpenRCT2
                 return false;
 
             char buffer[128];
-            while (feof(fp) == false)
+            while (!static_cast<bool>(feof(fp)))
```

Comment:
Not sure about this one.

Reply:
I, too, prefer the old style, but for the proposed solution is more in line with the C++ way. (Of course, better solution would be to not use a libc function.)

Reply:
`feof(fp) == 0`?